js/modal-big-picture.js

```diff
@@ -33,18 +33,18 @@ const renderPhoto = (photo) => {
 
 // функция для открытия окна
 export function openUserModal(picture) {
-  toggleClasses(true);
-  renderPhoto(picture);
-  renderComments(picture.comments);
+  toggleClasses(true); // меняем классы
+  renderPhoto(picture); // создаем фото
+  renderComments(picture.comments); // создаем комменты
```

Comment:
(они уже созданы, мы их просто отображем)
```suggestion
  renderComments(picture.comments); // рисуем комменты
```